samples/snippets/pom.xml

```diff
@@ -24,25 +24,20 @@
   </properties>
 
 
-  <!-- [START documentai_install_with_bom] -->
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>com.google.cloud</groupId>
-        <artifactId>libraries-bom</artifactId>
-        <version>4.4.0</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <dependencies>
+    <!-- [START documentai_install_without_bom] -->
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-document-ai</artifactId>
+      <version>0.1.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>1.103.1</version>
```

Comment:
If these samples/tests depend on google-cloud-storage, you will need to add this dependency to `samples/install-without-bom/pom.xml` and `samples/snapshot/pom.xml` as the same tests and samples are compiled in those modules.

Reply:
Some samples use Storage, Mike, can you put move this above to be included in the region tag and delete the version?

Reply:
Also need to add this below where the `</dependencies>` is closed
```
<!-- [START documentai_install_with_bom] -->
</dependencies>
<!-- [END documentai_install_with_bom] -->